### PR TITLE
Feat/58 artist progress popups

### PR DESCRIPTION
**Pull request by katjam**
Partial #58 

- Message pops up on 3 in a row incorrect, every 3 paris matched, end.

NOTE: The message is hardcoded right now. Need to add speech strings.

### CODE REVIEW

File: src/Main.elm [hunk 1]
```diff
@@ -227,6 +225,78 @@ getPair orphan allCards =
...
+
+            else
+                None
+    in
+    if progressType == None then
+        ( False, "" )
+
+    else
+        ( True
+        , Maybe.withDefault "" (List.head (getProgressSpeech progressType))
```
Comment (Nikomus) [inline src/Main.elm:296]: I think this is always picking the first item from the list of options in each category?

**Comment by Nikomus** [comment]
Thanks!
image

**Review by Nikomus — APPROVED** [review]
Look at shuffling the responses, but otherwise looks good!

**Comment by katjam** [comment]
Thanks. Shuffling the responses is the next step. It's not very straightforward in elm. :)

**Comment by katjam** [comment]
I'm still not happy with the css either. Some of the complete responses are too long to fit that small bubble + if people scale up their font size, the bubble won't cover it.

File: src/Main.elm (hunk 1 above)
Comment (katjam) [reply src/Main.elm:296]: Yup. It's a WIP. Good spot.

**Pull request merged**